src/stp/new_skills/GoToPos.cpp

```diff
@@ -26,7 +26,9 @@ Status GoToPos::onUpdate(const StpInfo &info) noexcept {
     // Set velocity and angle commands
     command.mutable_vel()->set_x(targetVelocity.x);
     command.mutable_vel()->set_y(targetVelocity.y);
-    command.set_w(robotCommand.angle.getAngle());
+
+    if(info.getAngle()) command.set_w(info.getAngle().getAngle());
+    else command.set_w(robotCommand.angle.getAngle());
```

Comment:
Do we really need to use info.getAngle().getAngle() ? It seems a bit weird to call getAngle() twice

Reply:
Yes, we need a double. stpInfo.getAngle() returns an Angle, Angle.getAngle() returns a double. I think the last getAngle() its naming is wrong, but that's in the Angle class so out of scope for this PR